lib/storage/table.go

```diff
@@ -162,21 +167,29 @@ func (tb *table) addPartitionNolock(pt *partition) {
 	tb.ptws = append(tb.ptws, ptw)
 }
 
+func (tb *table) waitForForceMergeCompletion() {
```

Comment:
Could we change `forceMergesRunning` to waitGroup and call `.Wait()` on table's `.MustClose()` instead?